### PR TITLE
Change negative to positive description

### PR DESCRIPTION
**Pull request by audreymaniez**
See #467

### CODE REVIEW

File: _rules/meta-viewport-b4f0c3.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ id: b4f0c3
 name: meta viewport does not prevent zoom
 rule_type: atomic
 description: |
-  This rule checks that the `meta` element is not used to block the user agent ability to zoom.
+  This rule checks that the `meta` element preserve the user agent ability to zoom.
```
Comment (WilcoFiers) [inline _rules/meta-viewport-b4f0c3.md]: Suggest "retains" instead of "preserves"

Comment (EmmaJP) [reply _rules/meta-viewport-b4f0c3.md]: https://wikidiff.com/retain/preserve - I think "preserves" is better.

Alternatively:
name: meta viewport is not preventing zoom
description: This rule checks that the `meta` element is not preventing the user agent ability to zoom.

Comment (EmmaJP) [reply _rules/meta-viewport-b4f0c3.md]: Or...
name: meta viewport is preserving zoom
description: This rule checks that the `meta` element is preserving the user agent ability to zoom.

Comment (audreymaniez) [reply _rules/meta-viewport-b4f0c3.md]: Thank you for your comments. I'm not en english native speaker I can't have an opinion. @WilcoFiers @Jym77 @jkodu what is your opinion: retains vs preserves?

Comment (Jym77) [reply _rules/meta-viewport-b4f0c3.md]: I have no preference.

**Comment by jkodu** [comment]
I think retains is better. Let us get this moving :)

**Comment by jkodu** [comment]
Merging this PR. If the description for this rule needs to be updated or discussed further, please open a new issue.

**Pull request merged**